presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -256,9 +275,12 @@ public Ssl()
         private static final Predicate<Properties> IF_SSL_ENABLED =
                 checkedPredicate(properties -> SSL.getValue(properties).orElse(false));
 
+        private static final Predicate<Properties> IF_SSL_VERIFICATION_ENABLED =
```

Comment:
Please define this in `SslVerification` and reuse it.

Reply:
Changed the code to define `IF_SSL_VERIFICATION_ENABLED` in `SslVerification` and reuse it.
Static import is not used in order to declare `SslVerification` as a private class (as other property classes are).